src/BookReaderComponent/BookReaderComponent.js

```diff
@@ -0,0 +1,112 @@
+/**
+ * BookReaderTemplate to load BookNavigator components
+ */
+
+import { LitElement, html, css } from 'lit-element';
+
+import '../ItemNavigator/ItemNavigator.js'
+import '../BookNavigator/BookNavigator.js'
+
+export class BookReader extends LitElement {
+  static get properties() {
+    return {
+      base64Json: { type: String },
+      baseHost: { type: String },
+    };
+  }
+
+  constructor() {
+    super();
+    this.base64Json = '';
+    this.baseHost = 'https://archive.org';
```

Comment:
Question; won't this override what the properties are set to? Or does this happen before and this is how defaults are set in Lit?

Reply:
Hi great questions, this is the default value - used if the property is not set 😊